python/orca/src/bigdl/orca/learn/metrics.py

```diff
@@ -54,11 +74,11 @@ def get_mxnet_metric(self):
         invalidInputError(False, "not implemented")
 
     @abstractmethod
-    def get_name(self):
+    def get_name(self) -> str:
         pass
 
     @staticmethod
-    def convert_metrics_list(metrics, backend="bigdl"):
+    def convert_metrics_list(metrics, backend: str = "bigdl"):
```

Comment:
What's the type of `metrics`?

Reply:
quite complicated, not target to add type here, this method is not for users